src/Engine/ProtoCore/Parser/AssociativeAST.cs

```diff
@@ -1663,6 +1663,7 @@ public IEnumerable<string> ReturnKeys
         public string ObsoleteMessage { get; protected set; }
         public bool IsObsolete { get { return !string.IsNullOrEmpty(ObsoleteMessage); } }
         public bool IsLacingDisabled { get; protected set; }
+        public bool AllowArrayPromotion { get; protected set; } = true;
```

Comment:
@aparajit-pratap should this also be added to the imperative AST methodAttributes class - does that exist?

Reply:
😆